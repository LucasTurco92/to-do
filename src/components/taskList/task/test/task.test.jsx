import { render } from '@testing-library/react';
import Task from '../task';

it('When it invokes the Task component, should render it properly when isNew', ()=>{

    const data = {
        description: 'description',
        id:'123 342 543 465',
        isNew: 'true'
    }

    const { container } = render(<Task data={data} />);
    
    expect(container).toMatchSnapshot();
});


it('When it invokes the Task component, should render it properly when isNew is false', ()=>{

    const data = {
        description: 'description',
        id:'123 342 543 465',
        isNew: 'false'
    }

    const { container } = render(<Task data={data} />);
    
    expect(container).toMatchSnapshot();
});