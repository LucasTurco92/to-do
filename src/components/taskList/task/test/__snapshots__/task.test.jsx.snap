// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When it invokes the Task component, should render it properly when isNew 1`] = `
<div>
  <div
    class="sc-gsDKAQ kpVBXQ"
  >
    <div
      class="sc-bdvvtL ivoDIz"
    />
    <input
      class="sc-hKwDye jYfISF"
      font-size="1em"
    />
  </div>
</div>
`;

exports[`When it invokes the Task component, should render it properly when isNew is false 1`] = `
<div>
  <div
    class="sc-gsDKAQ kpVBXQ"
  >
    <div
      class="sc-bdvvtL ivoDIz"
    />
    <input
      class="sc-hKwDye jYfISF"
      font-size="1em"
    />
  </div>
</div>
`;
